src/tools/genVersionHeader.pl

```diff
@@ -53,6 +57,7 @@
         my $hasmod = `darcs whatsnew --repodir="$opt_t" -l`;
         $opt_V .= '-dirty' unless $?;
     }
+    $cv = `darcs optimize --reorder; darcs changes --context | sort | md5sum`; # this is untested
```

Comment:
[`man darcs`](https://linux.die.net/man/1/darcs) for `optimize` states that it "... modifies the current repository ..." which doesn't seem like a good idea here.

I've no idea what the correct incantation would be, or even if there is one.  Darcs has a very difference conception of history.  Rather then guessing, how about just leaving this TODO?

```perl
    $cv = ""; # TODO
```